install/README.md

```diff
@@ -35,66 +35,71 @@ Once the CSM installation has completed, other product streams for the HPE Cray
     1. [Configure Prometheus Alert Notifications](#7-configure-prometheus-alert-notifications)
     1. [Update Firmware with FAS](#8-update-firmware-with-fas)
     1. [Prepare Compute Nodes](#9-prepare-compute-nodes)
-1. [Next Topic](#next-topic)
-1. [Troubleshooting Installation Problems](#troubleshooting-installation-problems)
+    1. [Next Topic](#10-next-topic)
+    1. [Troubleshooting Installation Problems](#troubleshooting-installation-problems)
```

Comment:
Since the troubleshooting link is under the install section, does this correctly imply that it does not include any info on troubleshooting stuff from the preinstall part?